gatsby/src/components/desktop-top-content/desktop-top-content.module.scss

```diff
@@ -1,3 +1,9 @@
+.desktopTopContent {
+  @media (min-width: 992px) and (min-height: 450px) {
```

Comment:
Please, use mixins instead of hardcoded values. See more e.g. here - https://github.com/cesko-digital/covid.gov.cz/blob/e284de52d769941c952772464ffdfecd0eb7f219/gatsby/src/layouts/desktop-left-menu-layout.module.scss#L12